node/pkg/watchers/ibc/watcher.go

```diff
@@ -0,0 +1,735 @@
+package ibc
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+	"github.com/certusone/wormhole/node/pkg/p2p"
+	"github.com/certusone/wormhole/node/pkg/readiness"
+	"github.com/certusone/wormhole/node/pkg/supervisor"
+	"github.com/certusone/wormhole/node/pkg/watchers/cosmwasm"
+	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+
+	gossipv1 "github.com/certusone/wormhole/node/pkg/proto/gossip/v1"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	"github.com/tidwall/gjson"
+
+	"nhooyr.io/websocket"
+	"nhooyr.io/websocket/wsjson"
+
+	ethCommon "github.com/ethereum/go-ethereum/common"
+
+	"go.uber.org/zap"
+)
+
+type (
+	// ChainConfig is the list of chains to be monitored over IBC, along with their channel data.
+	ChainConfig []ChainConfigEntry
+
+	// ChainConfigEntry defines the entry for chain being monitored by IBC.
+	ChainConfigEntry struct {
+		ChainID  vaa.ChainID
+		MsgC     chan<- *common.MessagePublication
+		ObsvReqC <-chan *gossipv1.ObservationRequest
+	}
+)
+
+var (
+	// Chains defines the list of chains to be monitored by IBC. Add new chains here as necessary.
+	Chains = []vaa.ChainID{vaa.ChainIDTerra2}
+
+	// Features is the feature string to be published in heartbeat messages. It will include all chains that are actually enabled on IBC.
+	Features = ""
```

Comment:
Are heartbeat messages implemented?
Is this var access from a different file?

Reply:
Updated the comment to say "gossip heartbeat messages", which is where this is referenced.